docs/integrations/system_metrics.md

```diff
@@ -1,28 +1,68 @@
-By default, **Logfire** does not collect system metrics.
+The [`logfire.instrument_system_metrics()`][logfire.Logfire.instrument_system_metrics] method can be used to collect system metrics with **Logfire**, such as CPU and memory usage.
 
-To enable metrics, you need to install the `logfire[system-metrics]` extra:
+## Installation
+
+Install `logfire` with the `system-metrics` extra:
 
 {{ install_logfire(extras=['system-metrics']) }}
 
-### Available Metrics
-
-Logfire collects the following system metrics:
-
-* `system.cpu.time`: CPU time spent in different modes.
-* `system.cpu.utilization`: CPU utilization in different modes.
-* `system.memory.usage`: Memory usage.
-* `system.memory.utilization`: Memory utilization in different modes.
-* `system.swap.usage`: Swap usage.
-* `system.swap.utilization`: Swap utilization
-* `system.disk.io`: Disk I/O operations (read/write).
-* `system.disk.operations`: Disk operations (read/write).
-* `system.disk.time`: Disk time (read/write).
-* `system.network.dropped.packets`: Dropped packets (transmit/receive).
-* `system.network.packets`: Packets (transmit/receive).
-* `system.network.errors`: Network errors (transmit/receive).
-* `system.network.io`: Network I/O (transmit/receive).
-* `system.network.connections`: Network connections (family/type).
-* `system.thread_count`: Thread count.
-* `process.runtime.memory`: Process memory usage.
-* `process.runtime.cpu.time`: Process CPU time.
-* `process.runtime.gc_count`: Process garbage collection count.
+## Usage
+
+```py
+import logfire
+
+logfire.configure()
+
+logfire.instrument_system_metrics()
+```
+
+Then in your project, click on 'Dashboards' in the top bar, click 'New Dashboard', and select 'Basic System Metrics (Logfire)' from the dropdown.
+
+## Configuration
+
+By default, `instrument_system_metrics` collects only the metrics it needs to display the 'Basic System Metrics (Logfire)' dashboard. You can choose exactly which metrics to collect and how much data to collect about each metric. The default is equivalent to this:
+
+```py
+logfire.instrument_system_metrics({
+    'system.cpu.simple_utilization': None,
+    'system.memory.utilization': ['available'],
+    'system.swap.utilization': ['used'],
+})
+```
+
+To collect lots of detailed data about all available metrics, use `logfire.instrument_system_metrics(base='full')`. This is equivalent to:
```

Comment:
Maybe good to have a warning here saying roughly how many metrics this will generate, and that that might be expensive if run full time, especially on multiple machines.